Singer.API/ClientApp/src/app/modules/core/services/singer-location-api/singer-location.proxy.ts

```diff
@@ -32,13 +32,11 @@ export class SingerEventLocationProxy {
       return this.apiService.get('api/eventlocation', searchParams).pipe(map(res => res));
    }
 
-   updateSingerEventLocation(id: string, updateSingerEventLocationDTO: UpdateSingerEventLocationDTO) {
-      return this.apiService.put(`api/eventlocation/${id}`, updateSingerEventLocationDTO).pipe(map(res => res));
+   updateSingerLocation(id: string, updateSingerLocationDTO: UpdateSingerLocationDTO) {
+      return this.apiService.put(`api/eventlocation/${id}`, updateSingerLocationDTO).pipe(map(res => res));
```

Comment:
I know we talked about the `EventLocation` thing, but shouldn't it be `Location` in the URL?